lib/consts/global_style.dart

```diff
@@ -6,7 +6,7 @@ class AppColor {
   static const Color secondary = Color(0xFF324ED9);
   static const Color lightGray = Color(0xFFA7ABB8);
   static const Color gray = Color(0xFF666666);
-  static const Color darkBlue = Color(0xFF3D3541);
+  static const Color darkBlue = Color(0xFF242D58);
```

Comment:
This has same value as `onyx`. Maybe we can use `onyx` na lang for this value.

Reply:
@AubreyV Actually they were interchanged ma'am, onyx is 3D3541 and dark blue is 242D58

Reply:
@rdelrosariojr oki pls apply this. si onyx dapat `3D3541` and dark blue is `242D58`

Reply:
@rdelrosariojr this has been fixed in the [[UI] Implemented tab layout with dynamic content #16 PR](https://github.com/framgia/sph_flutter-capstone/pull/16/files#diff-4451a9249b8acf87752363610d509d955736eccb1b9550071b34c360b1bb6b68)